e2e/rbd.go

```diff
@@ -2713,8 +2713,8 @@ var _ = Describe("RBD", func() {
 					e2elog.Failf("failed to delete storageclass with error %v", err)
 				}
 				err = createRBDStorageClass(f.ClientSet, f, defaultSCName, nil, map[string]string{
-					"mapOptions":   "lock_on_read,queue_depth=1024",
-					"unmapOptions": "force",
+					"mapOptions":   "krbd:lock_on_read,queue_depth=1024",
+					"unmapOptions": "krbd:force",
```

Comment:
remove kbd here to make sure backward compatibility is tested?

Reply:
That should work, yes I can remove it.